src/ts/team.ts

```diff
@@ -95,9 +95,14 @@ document.addEventListener('DOMContentLoaded', () => {
   if (document.getElementById('itemSelect')) {
     eventBackgroundColor = (document.getElementById('itemSelect') as HTMLSelectElement).selectedOptions[0].dataset.color;
   }
+  const weekends = !!document.getElementById('showWeekends');
```

Comment:
instead, use a data attribute on element `#scheduler`. `<div id='scheduler'  data-weekends='{{ App.Users.userData.show_weekends }}' ...`

and grab this from `calendarEl.dataset` in team.ts.